### PR TITLE
Implement Matomo tracker

### PR DESCRIPTION
**Pull request by KevinBoulongne**
Blocked by https://github.com/Infomaniak/android-core/pull/15

### CODE REVIEW

File: app/src/main/java/com/infomaniak/mail/ui/LaunchActivity.kt [hunk 1]
```diff
@@ -32,18 +33,21 @@ class LaunchActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         lifecycleScope.launch(Dispatchers.IO) {
-            if (AccountUtils.requestCurrentUser() == null) loginUser() else startApp()
+            AccountUtils.requestCurrentUser()
+                .also { trackUserId(AccountUtils.currentUserId) }
+                ?.let { startApp() }
+                ?: run { loginUser() }
```
Comment (JorisBodin) [inline app/src/main/java/com/infomaniak/mail/ui/LaunchActivity.kt:39]: It's really not that clear of an "if".

File: app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt [hunk 2]
```diff
@@ -230,6 +231,7 @@ class MenuDrawerFragment : Fragment() {
     fun onDrawerOpened() {
         canNavigate = true
         mainViewModel.forceRefreshMailboxes()
+        requireActivity().trackScreen(this::class.java.name, this::class.java.simpleName)
```
Comment (JorisBodin) [inline app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt:234]: Just Fragment.trackScreen() is not enough?

**Review by JorisBodin — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/infomaniak/mail/ui/LaunchActivity.kt (hunk 1 above)
Comment (KevinBoulongne) [reply app/src/main/java/com/infomaniak/mail/ui/LaunchActivity.kt]: Done.

File: app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt (hunk 2 above)
Comment (KevinBoulongne) [reply app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt]: Done.

**Review by JorisBodin — APPROVED** [review]

**Pull request merged**